course/steps/08_react-apollo/en.md

```diff
@@ -49,20 +49,20 @@ The **Apollo Client** lib offers native integration with React, through _hooks_.
     ```diff
     // react/Countdown.tsx
     import React from 'react'
+    ...
     +import { useQuery } from 'react-apollo'
-    +import useProduct from 'vtex.product-context/useProduct'
 
-    import { useCssHandles } from 'vtex.css-handles'
+    +import useProduct from 'vtex.product-context/useProduct'
 
-    +import productReleaseDateQuery from './queries/productReleaseDate.graphql'
+    +import productReleaseDate from './queries/productReleaseDate.graphql'
     ```
     > It is important to higlight that there is the possibility of your IDE showing an error while importing `product-context`.
 
-4.  Define the query using the `productReleaseDateQuery` importaded and the `useQuery`, using the `useProduct()` data. As mentioned before, both are  hooks, which means that they must be added inside a React functional component. In this case, `Countdown`.
+4.  Define the query using the `productReleaseDate` importaded and the `useQuery`, using the `useProduct()` data. As mentioned before, both are  hooks, which means that they must be added inside a React functional component. In this case, `Countdown`.
```

Comment:
```suggestion
4.  Define the query using the `productReleaseDate` imported and the `useQuery` hook, you can find the product data in `useProduct` hook. Since they are (hooks)[https://reactjs.org/docs/hooks-intro.html], they only work inside react functional components. 
```